src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -18,6 +18,14 @@ class DataExplorerVisualizations extends React.Component {
     super(props);
     this.state = {
       manifestEntryCount: 0,
+      exportInProgress: false,
+      exportFileName: null,
```

Comment:
do we need this variable? we are designating a filename in the configuration and also storing one given back by the manifest service?